developer_guides/topology/topology.rst

```diff
@@ -292,6 +292,45 @@ where:
   can be chosen predefined configurations such as MONO_PDM0_MICA,
   STEREO_PDM0, FOUR_CH_PDM0_PDM1 etc.
 
+.. _dsp-core-in-topology:
+
+1.7 DSP Core Index
+------------------
+
+We can specify a DSP Core Index that a pipeline or component will be scheduled
+on in the topology file.
```

Comment:
The topology file can specify on which DSP core a pipeline or component will be scheduled on.